src/CabanaPD_ForceModels.hpp

```diff
@@ -71,6 +72,45 @@ struct BaseTemperatureModel
     }
 };
 
+// This class stores temperature parameters needed for heat transfer, but not
+// the temperature itself (stored instead in the static temperature class
+// above).
+struct BaseDynamicTemperatureModel
+{
+    double delta;
+
+    double thermal_coeff;
+    double kappa;
+    double cp;
+    bool constant_conductivity;
```

Comment:
I guess this refers to the "micro-conductivity"? Perhaps call it "constant_microconductivity"?

Reply:
Changed